src/core/AvatarClient.ts

```diff
@@ -79,6 +79,14 @@ export class AvatarClient extends HTTPClient {
     return this.connect(avatarId);
   }
 
+  enableConversationalMode() {
+    navigator.mediaDevices.getUserMedia({ audio: true }).then(() => {
+      if (this.isConnected) {
+        this.room?.localParticipant?.setMicrophoneEnabled(true);
+      }
+    });
+  }
```

Comment:
nitpick: use `async`/`await` instead of `then`. Also use `try`/`catch` to avoid unhandled rejected promise error